### DISCUSSION TITLE
recipes/desserts/cake/hummingbird-cake/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#998] [Comments]
# recipes/desserts/cake/hummingbird-cake/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/cake/hummingbird-cake/

**Comment by nicholaswilde** [comment]
Try with brown butter cream cheese frosting and a whole vanilla bean.
Try with pineapple compote instead of crushed pineapple.

**Reply by nicholaswilde** [reply]
2/3 the batter to make 2 tiers
Add more frosting in between layers
Bottom layer should have flat side down instead of dome

**Reply by nicholaswilde** [reply]
Tried with brown butter cream cheese frosting
